ocs_ci/ocs/cluster.py

```diff
@@ -1273,11 +1273,11 @@ def check_osd_tree_1az_vmware(osd_tree, number_of_osds):
     return check_osds_in_hosts_osd_tree(all_hosts_flatten, osd_tree)
 
 
-def check_osd_tree_3az_aws(osd_tree, number_of_osds):
+def check_osd_tree_3az(osd_tree, number_of_osds):
```

Comment:
The name `check_osd_tree_3az_cloud` will be more accurate here since the function is for testing the cloud platforms.

Reply:
Done